.flake8

```diff
@@ -0,0 +1,26 @@
+[flake8]
+max-line-length=120
+application_import_names=bot
+docstring-convention=all
+ignore=
+    P102,B311,W503,E226,S311,
+    # Missing Docstrings
+    D100,D104,D105,D107,
+    # Docstring Whitespace
+    D203,D212,D214,D215,
+    # Docstring Quotes
+    D301,D302,
+    # Docstring Content
+    D400,D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417,
+    # Type Annotations
+    ANN002,ANN003,ANN101,ANN102,ANN204,ANN206,
+    # Binary operators over multiple lines
+    W504,
+exclude=
+    __pycache__,.cache,
+    venv,.venv,
+    tests,
```

Comment:
I'm not sure if we still want to have tests, but even so, wouldn't it be fine to run linting on it anyways?